builder-create-dokku-image

```diff
@@ -48,6 +48,11 @@ hook-apt-builder-create-dokku-image() {
   eval "ARG_ARRAY=($DOCKER_ARGS)"
 
   COMMAND="$(fn-apt-command "$APP" "$DOKKU_IMAGE" "/tmp/apt")"
+
+  if [[ "$UPDATE_CA_CERTS" == "1" ]]; then
+      COMMAND="update-ca-certificates && $COMMAND"
```

Comment:
Why not make this something arbitrary? As in, make it such that someone can specify a command via an environment variable and we inject it [here](https://github.com/dokku-community/dokku-apt/blob/master/internal-functions#L61)? Otherwise, its fairly limited in scope and adds the potential for us to add more "flags" in the future.